main.py

```diff
@@ -1,10 +1,39 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_info["race"].get("name"),
+            description=player_info["race"].get("description"),
+        )
+
+        for skill in player_info["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race
+            )
+
+        guild, _ = Guild.objects.get_or_create(
+            name=player_info["guild"].get("name"),
+            description=player_info["guild"].get("description")
+        ) if player_info["guild"] else (None, False)
+
+        Player.objects.create(
```

Comment:
Consider wrapping `Player.objects.create` in a try-except block to handle potential validation errors gracefully, such as unique constraint violations.